doc/smart-switch/reboot/reboot-hld.md

```diff
@@ -89,12 +90,12 @@ DPUs are internally connected to the NPU via PCI-E bridge. Below is the reboot s
 service, continuing until the timeout is reached. Once the DPU successfully terminates all services, it responds to the gNOI RebootStatus RPC with STATUS_SUCCESS and 'active'
 will be set to false in the RebootStatusResponse. Until the services are terminated gracefully, 'active' will be '1' in the RebootStatusResponse.
 
-* Subsequently, the NPU detaches the DPU PCI device using the sysfs interface by executing `echo 1 > /sys/bus/pci/devices/XXXX:XX:XX.X/remove`. This detachment is currently performed via sysfs, with future plans to implement a vendor-specific API when the DPU bus information cannot have a fixed value.
+* Subsequently, the NPU detaches the DPU PCI device via the pci_detach() vendor API, or through the sysfs interface by executing echo 1 > /sys/bus/pci/devices/XXXX:XX:XX.X/remove, if the vendor API is unavailable.
 
 * Next, the NPU triggers a platform vendor reboot API to initiate the reboot process for the DPU. If the DPU is stuck or unresponsive, the DPU reboot platform API should
 attempt a cold boot or power cycle to recover it.
 
-* The NPU either immediately rescans the PCI bus upon return or after a specified timeout period. This rescan is performed via the sysfs interface by echoing '1' to /sys/bus/pci/rescan.
+* The NPU either immediately rescans the PCI bus upon return or after a specified timeout period. This rescan is performed via the pci_reattach() vendor API or, if the vendor API is unavailable, by echoing '1' to /sys/bus/pci/rescan.
```

Comment:
@vvolam how is the timeout specified?

Reply:
@prgeor There is no timeout here. Rescan happens after reboot API returns the handle. I will reword HLD accordingly. Thank you!